node_exporter.go

```diff
@@ -110,6 +111,21 @@ func loadCollectors(list string) (map[string]collector.Collector, error) {
 		}
 		collectors[name] = c
 	}
+	for name, enabled := range collectorsEnabled {
```

Comment:
Yes, currently this is the case for maximum compatibility with the old way. We could also go for being able to disable, but when I wrote this, I didn't come up with a satisfying idea how to marry both versions without possibly behaving weird, i.e. you set collectors via the old way and combine that with the new flag, what should happen then? We would probably just need to define the behavior, but I didn't want to come up here with something without having discussed this first.